inttest/footloose-alpine/Dockerfile

```diff
@@ -1,7 +1,32 @@
 ARG ALPINE_VERSION
+ARG GOLANG_IMAGE
+
+# troubleshoot doesn't publish aarch32 binaries, so it's required to build them on the go...
+FROM $GOLANG_IMAGE as supportbundle
+
+ARG ARCH
+ENV TROUBLESHOOT_VERSION=v0.59.0
+
+RUN mkdir /out \
+  && if [ "$ARCH" = arm ]; then \
```

Comment:
> I think the reasonable way of doing this is modifying github actions so that it releases also ARM architecture. Checking the code doesn't seem like a very big change. I'll have a look at this today.

I've also had a look into this. The question is if upstream wants to support this. They're using goreleaser, so it needs to be decided to ship binaries for which ABI and so on. I've played around with the GitHub workflows and everything kinda worked. Either way, it's their decision.

> Is this something that you need merged REALLY quick or can we wait a few days and see if we get troubleshoot released?

It's only an issue if the basic inttest fails on non x86_64 platforms and one needs to figure out what failed. I currently have it in the k8s 1.27 WIP branch, where it can stay until the dust settles there. So no pressure on this.

Reply:
Okay. No longer necessary with troubleshoot v0.61.0.